code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -19,6 +21,12 @@
 		src.pixel_x = rand(-5, 5) * PIXEL_MULTIPLIER
 		src.pixel_y = rand(-5, 5) * PIXEL_MULTIPLIER
 
+/proc/get_special_fruits(var/filter=HYDRO_PREHISTORIC|HYDRO_VOX)
+	. = list()
+	for(var/type in existing_typesof(/obj/item/weapon/reagent_containers/food/snacks/grown))
```

Comment:
You can cast this variable to `/obj/item/weapon/reagent_containers/food/snacks/grown` and then you'll be able to remove the `:` below.